src/main/java/com/fibermc/essentialcommands/commands/NearCommand.java

```diff
@@ -0,0 +1,67 @@
+package com.fibermc.essentialcommands.commands;
+
+import java.util.Objects;
+
+import com.fibermc.essentialcommands.EssentialCommands;
+import com.fibermc.essentialcommands.playerdata.PlayerData;
+import com.fibermc.essentialcommands.text.ECText;
+import me.drex.vanish.api.VanishAPI;
+
+import com.mojang.brigadier.Command;
+import com.mojang.brigadier.arguments.IntegerArgumentType;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.server.command.ServerCommandSource;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.text.MutableText;
+import net.minecraft.text.Text;
+import net.minecraft.util.math.Vec3d;
+
+public class NearCommand implements Command<ServerCommandSource> {
```

Comment:
should this be in the `utility` package since it doesn't rely on any persistent mod state? (Aside from Vanish, which I think is more of an integration than part of the feature's primary purpose, if that makes sense)

Reply:
yeah sounds good